Composer/packages/client/src/pages/publish/LogDialog.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { Dialog } from 'office-ui-fabric-react/lib/Dialog';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+export interface LogDialogProps {
+  value?: string;
```

Comment:
It seems like the caller always provides a value, in fact it only shows this dialog when the value is not null, why is this optional then?

Reply:
cancel `?`